cli/wallet/multisig.go

```diff
@@ -91,6 +106,81 @@ func signMultisig(ctx *cli.Context) error {
 	return nil
 }
 
+func sendTransaction(ctx *cli.Context) error {
+	data, err := ioutil.ReadFile(ctx.String("in"))
+	if err != nil {
+		return cli.NewExitError(fmt.Errorf("can't read input file: %v", err), 1)
+	}
+	txSig := new(TxWithSignatures)
+	if err := json.Unmarshal(data, txSig); err != nil {
+		return cli.NewExitError(fmt.Errorf("can't parse transaction: %v", err), 1)
+	}
+
+	w := new(transaction.Witness)
+	script, err := hex.DecodeString(txSig.Script)
+	if err != nil {
+		return cli.NewExitError(err, 1)
+	}
+	pubs, err := parseScript(script)
```

Comment:
Why do need to parse the script?

Reply:
We need to extract public keys from it to push signatures in the sorted order.

Reply:
But don't you already have them in `Sigs` map?

Reply:
I have, but they are unordered there.

Reply:
Well, to me it's still easier to get them from it, sort and use in whatever fashion you need. Parsing bytecode is not something you should be doing here.